README.md

```diff
@@ -15,6 +15,7 @@ Import guides are hosted on the [official Obsidian Help site](https://help.obsid
 - [Import from Google Keep](https://help.obsidian.md/import/google-keep)
 - [Import from Microsoft OneNote](https://help.obsidian.md/import/onenote)
 - [Import from Notion](https://help.obsidian.md/import/notion)
+- [Import from Notion API(WIP)](https://help.obsidian.md/import/notion-api)
```

Comment:
Use `import/notion` (we won't have a separate page)

Reply:
done, just remove it. https://github.com/obsidianmd/obsidian-importer/pull/444/commits/9a03fc09d94daa989236829c49d25e98b6975843